lib/launcher.ts

```diff
@@ -251,7 +254,7 @@ let initFn = function(configFile: string, additionalConfig: Config) {
         let createNextTaskRunner = () => {
           let task = scheduler.nextTask();
           if (task) {
-            let taskRunner = new TaskRunner(configFile, additionalConfig, task, forkProcess);
```

Comment:
Right, I missed that. `additionalConfig` might as well be renamed to make it something more clear.